cpp/tests/utilities/column_utilities.cu

```diff
@@ -1038,13 +1038,26 @@ struct column_view_printer {
     if (col.is_empty()) return;
     auto h_data = cudf::test::to_host<std::string>(col);
 
+    // explicitly replace '\r' and '\n' characters with "\r" and "\n" strings respectively.
+    auto cleaned = [](std::string const& in) {
```

Comment:
Maybe `replace_special_chars` is more expressive than just `cleaned`?

Reply:
```suggestion
    auto cleaned = [](std::string_view in) {
```